java/piranha/src/test/resources/config/properties_test_receive.json

```diff
@@ -0,0 +1,19 @@
+{
+  "piranhaMethodProperties": [
+    {
+      "methodName": "isToggleDisabled",
+      "flagType": "control",
+      "receiverType": "com\\.uber\\.piranha\\.XPTest"
+    },
+    {
+      "methodName": "isFlagTreated",
+      "flagType": "treated",
+      "receiverType": "com.uber.*"
```

Comment:
where is this used?

Reply:
`properties_test_receive.json` and `properties_test_return.json` are used in `XPFlagCleanerTest`

If you were asking about this particular testcase:
`caseWithFlagNameAsStringLiteralWithReceiverTypeRegex`

Reply:
i was asking for this specific case. thanks.